delighted/client.py

```diff
@@ -27,15 +28,16 @@ def __init__(self, api_key=None, api_base_url=None, http_adapter=None):
                              "delighted.api_key = '123abc' or passing " +
                              "api_key='abc123' when instantiating client.")
 
-    def request(self, method, resource, headers={}, params={}):
+    def request(self, method, url, headers={}, params={}, full_url=False):
```

Comment:
Most of our APIs are using relate url within this client, while the `Link` header is a full absolute URL.